packages/design-system/src/components/Badge/Badge.spec.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import { BadgeDropdown, BadgePopover, BadgeTag, BadgeValue } from '.';
+
+/* eslint-disable testing-library/prefer-screen-queries */
+
+context('<Tag />', () => {
+	const label = 'Delightful';
+	const items = [
+		{ id: '1', label: 'Component', onClick: () => {} },
+		{ id: '2', label: 'Item', onClick: () => {} },
+	];
+
+	it('should render BadgeTag', () => {
+		cy.mount(<BadgeTag label={label} />);
+
+		cy.getByTestId('badge-label').should('have.text', label);
+	});
+
+	it('should render BadgeValue', () => {
+		cy.mount(<BadgeValue label={label} value={items.map(v => v.label)} withOperator={false} />);
+
+		cy.getByTestId('badge-label').should('have.text', label);
+		cy.getByTestId('badge-divider');
+
+		items.map(v => v.label).map(v => cy.getByTestId(`badgevalue-${v}`).should('have.text', v));
+	});
+
+	it('should render BadgeDropdown', () => {
```

Comment:
don't you want to have interaction tests to test dropdown state ?

Reply:
Not really, because i use dropdown component which is in charge of user interaction so it has to be tested in dropdown unit tests and not in this component.
If dropdown behaviour change it requries us to update tests in this component which is not responsible of it.